app/uk/gov/hmrc/fileupload/controllers/EnvelopeController.scala

```diff
@@ -27,27 +29,41 @@ import uk.gov.hmrc.fileupload.utils.JsonUtils.jsonBodyParser
 import uk.gov.hmrc.fileupload.write.envelope._
 import uk.gov.hmrc.fileupload.write.infrastructure._
 import uk.gov.hmrc.fileupload.{EnvelopeId, FileId, _}
-import uk.gov.hmrc.play.microservice.controller.BaseController
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.postfixOps
 
+
+package object envelope {
+
+  import play.api.libs.concurrent.Execution.Implicits._
+
+  object EnvelopeController extends EnvelopeController(
+    withBasicAuth = MicroserviceGlobal.withBasicAuth,
```

Comment:
this was by necessity rather than design, the impl packages objects were the desired names. However we can't define the more than one package object with the same name. Since the controller objects were sitting in the same files as the respective classes this meant that we'd have 3 definitions of uk.gov.hmrc.fileupload.controllers.impl for Envelope, Event & File Controllers. So I had to pick a unique name for each one. 

Having said that the controller objects in their current state are not ideal and since I'm considering moving them all into MicroserviceGlobal.scala we can tidy up the names when we do that.